index.html

```diff
@@ -148,13 +151,31 @@ <h1 id="timeDisplay">03:00</h1>
       let currentDay = 1;
       let isDawn = true;
 
-      function displayTimeLeft(seconds) {
+      function formatTime(seconds) {
         const minutes = Math.floor(seconds / 60);
         const remainingSeconds = seconds % 60;
         const formattedMinutes = minutes < 10 ? "0" + minutes : minutes;
         const formattedSeconds =
           remainingSeconds < 10 ? "0" + remainingSeconds : remainingSeconds;
-        timeDisplay.textContent = `${formattedMinutes}:${formattedSeconds}`;
+        return `${formattedMinutes}:${formattedSeconds}`;
+      }
+
+      function parseTime(timeValue) {
+        if (/^\d*\.?\d+$/.test(timeValue)) {
+          parsedTime = Math.round(timeValue * 60);
+        } else if (timeValue.includes(':')) {
+          timeValue = timeValue.split(':');
+          parsedTime = timeValue[0] * 60 + timeValue[1] * 1;
+        } else if (timeValue.endsWith('s')) {
+          parsedTime = Math.round(timeValue.substr(0, timeValue.length - 1));
+        } else {
+          parsedTime = 0;
+        }
+        return parsedTime;
+      }
+
+      function displayTimeLeft(seconds) {
+        timeDisplay.textContent = formatTime(seconds)
 
```

Comment:
This is perfect! I think it would be great to restrict the user from entering letters and high numbers, something like this:

`if (isNaN(userInput)) {
        alert("Please enter a valid number.");
        return;
    }
    var number = parseInt(userInput);
    if (number > 1000) { // Assuming 1000 is the limit
        alert("The number is too high. Please enter a number less than 1000.");
        return;
    }`

This is something we can add later, as you mentioned.